giraph-core/src/main/java/org/apache/giraph/job/JobProgressTracker.java

```diff
@@ -43,13 +43,14 @@
   void logInfo(String logLine);
 
   /**
-   * Call this when you want to log an error line from any mapper to command
-   * line
+   * Call this when you want to log an error line and exception
+   * object from any mapper to command line
    *
    * @param logLine Line to log
+   * @param exByteArray Exception byte array
```

Comment:
Nit: add a comment here that KryoWritableWrapper.tryConvertFromByteArray can be used to get exception object back. (if this was java 8 I'd suggest having another method which takes Exception parameter and have default implementation of this one calling it, but it's not :-( )

Reply:
Done.